AppConnectSwift/SecurityQuestionViewController.swift

```diff
@@ -2,31 +2,47 @@ import UIKit
 
 class SecurityQuestionViewController: UIViewController, UITableViewDelegate, UITableViewDataSource  {
    
-    var tableDataSource: [String]  = ["What year were you born?",
-        "Last four digits of SSN or Tax ID number?",
-        "What is your fathers middle name?",
-        "What was the name of your first school?",
-        "Who was your childhood hero?",
-        "What is your favorite pastime?",
-        "What is your all-time favorite sports team?",
-        "What is your high school team or mascot?",
-        "What make was your first car or bike?",
-        "What is your pets name?",
-        "What is your mothers middle name?"
-    ]
+    var tableDataSource = [String]()
     var securityQuestion = "What year were you born?"
-    var securityQuestionID = 1
+    var securityQuestionID = -1
     var createAccountViewController = CreateAccountViewController()
     var userEmail : String!
     var userPassword : String!
     
+    var securityIdsByQuestion = [String : Int]()
+    
     @IBOutlet weak var tableView: UITableView!
     
     override func viewDidLoad() {
         super.viewDidLoad()
         tableView.delegate = self
         tableView.dataSource = self
         self.tableView.registerClass(UITableViewCell.self, forCellReuseIdentifier: "Cell")
+        
+        let clientFactory = MDClientFactory.sharedInstance()
+        let client = clientFactory.clientOfType(MDClientType.Hybrid)
+        
+        client.loadSecurityQuestionsWithCompletion() { (questions: [NSObject : AnyObject]!, error: NSError!) -> Void in
+            if error != nil {
+                NSOperationQueue.mainQueue().addOperationWithBlock({
+                    self.showDialog("Error", message: "There was an error retrieving the security questions", completion: nil)
+                });
+                
+                return
+            }
+            
+            for questionId in questions.keys {
```

Comment:
I think since we the dictionary comes directly from the iMedidata JSON, it'll actually be a string - looks like this does the trick:

```swift
for (questionId, question) in questions as! [String : String] {
    self.securityIdsByQuestion[question] = Int(questionId)
    self.tableDataSource.append(question)
}
```